css/card.css

```diff
@@ -66,28 +283,104 @@
     "head  head"
     "card  text"
     "card  text";
+  overflow-y: auto;
 }
 
 [role="dialog"][aria-hidden="true"] {
   display: none;
 }
 
+[role="dialog"][aria-hidden="noshow"] {
+  opacity: 0;
+}
+
+@media screen and (max-width: 1400px) {
+  #gacha-controls tcg-card {
+    --card-width: 300px;
+    --card-height: 419px;
+  }
+
+  .details-dialog-card {
+    width: 25vw;
+    height: calc(25vw*1.395);
+  }
+}
+
 @media screen and (max-width: 1000px) {
+
+  #gacha-controls tcg-card {
+    --card-width: 200px;
+    --card-height: 279px;
+  }
+
   #card-details-dialog {
     grid-template-areas:
       ".     x"
       "head  head"
       "text  text"
       "card  card";
-    overflow: auto;
+    overflow-y: auto;
+    overflow-x: hidden;
```

Comment:
I agree. I think leaving it at `overflow: auto` is sufficient. afaik, there shouldn't be any elements that are causing the dialog container to expanding past its defined width.

Reply:
That's fine - there was actually a horizontal scroll coming up when resizing the window down; and I'm not sure why; so I just added that. But since you are updating further the layout etc I can remove.